conda/conda_build_config.yml

```diff
@@ -0,0 +1,3 @@
+# bipedal-locomotion-framework requires 10.13 for std::bad_cast_any
+MACOSX_DEPLOYMENT_TARGET:      # [osx]
```

Comment:
If you use this, make sure to add the `__osx` run dependency: https://conda-forge.org/docs/maintainer/knowledge_base.html

Reply:
Done in https://github.com/robotology/robotology-superbuild/pull/652/commits/05593fddf73fac940540d6d48e1b181e11b7351d .